project/profiles/capacity4more/modules/c4m/content/c4m_content_wiki_page/c4m_content_wiki_page.features.field_base.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * @file
+ * c4m_content_wiki_page.features.field_base.inc
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function c4m_content_wiki_page_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_c4m_editing_open'.
+  $field_bases['field_c4m_editing_open'] = array(
```

Comment:
We used to keep track of this in a simple custom table.
I think we only need to retrieve this value on the page where we can change the settings from private to open, so it might be a bit overkill to add it as a field instance, no?